week1_KimChaeHyun/src/components/tab/FiterTab.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styles from './FilterTab.module.css';
+import { IcTrend, IcRec, IcFeed, IcMore } from "@assets/svgs";
+import DropdownMenu from '@components/dropdownMenu/DropdownMenu';
+
+const tabs = [
+  { id: 1, label: '트렌딩', icon: <IcTrend />, path: '/trending/week' },
+  { id: 2, label: '최신', icon: <IcRec />, path: '/recent' },
+  { id: 3, label: '피드', icon: <IcFeed />, path: '/feed' },
+];
+
+const FilterTab = () => {
+  const [activeTab, setActiveTab] = useState(1);
+  const navigate = useNavigate();
+
+  const handleTabClick = (tab) => {
+    setActiveTab(tab.id);
+    navigate(tab.path);
+  };
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+
+  return (
```

Comment:
일단 여기도 div 지옥이네요 채현씨.. 여기는 nav태그로 감싸줘도 될것 같고, 왜인진 모르지만 실제 벨로그에서는 전체 감싸주는건 div로 하고 그 중 왼쪽 탭 섹션만을 nav 태그로 감싸줬더라구요? 아무튼 시멘틱 태그를 적극 사용해봅시다~

Reply:
이렇게 해도 될진 모르겠는데 저라면 

```suggestion
  <nav>
    <section></section>
    <section></section>
  </nav>
```
이런식으로 할 거 같아요